docs/site/src/pages/reference/typescript-client/read-data.mdx

```diff
@@ -68,15 +68,18 @@ For more information on the `Frame` class, see [Frames](series-and-frames#frames
 
 While the above methods will cover most use cases, there are situations where
 it's necessary to query large volumes of data. Iterators provide a way to
-efficiently process data in chunks:
+efficiently process data in chunks. To configure reading in chunks, pass in the desired
+number of samples per iteration to the `openIterator` method. If unspecified,
+this value is `500,000` samples.
 
 ```typescript
 const start = TimeStamp.now();
 const end = start.add(TimeStamp.seconds(10));
 
 const iterator = await client.openIterator(
-    { start, end }, 
-    ["temperature", "humidity"]
+    { start, end },
+    ["temperature", "humidity"],
+    100, // chunkSize
```

Comment:
You wouldn't need the extra comment here if you use a TS options pattern instead. See my comment on the method signature